denonavr/denonavr.py

```diff
@@ -29,6 +31,11 @@
 AVR_X = ReceiverType(type="avr-x", port=80)
 AVR_X_2016 = ReceiverType(type="avr-x-2016", port=8080)
 
+DiscriptionType = namedtuple('DiscriptionType', ["port", "url"])
```

Comment:
Spelling: should this be "DescriptionType" and otherwise change discription to description?

Reply:
thanks for pointing this out, just fixed it